File: app/src/main/java/com/example/whattodo/WhatToDo.java
package com.example.whattodo;

import android.app.Application;
import android.app.NotificationChannel;
import android.app.NotificationManager;
import android.os.Build;

public class WhatToDo extends Application {
    public static final String MAINCHANNEL_ID = "Main Chanel";

    @Override
    public void onCreate() {
        super.onCreate();

        createNotificationChannels();
    }

    private void createNotificationChannels() {
        if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.O){
            NotificationChannel mainChannel = new NotificationChannel(
                    MAINCHANNEL_ID,
                    "Main Chanel",
                    NotificationManager.IMPORTANCE_HIGH
            );
            mainChannel.setDescription("This is MainChannel");

            NotificationManager manager = getSystemService(NotificationManager.class);
            manager.createNotificationChannel(mainChannel);

        }
    }
}

File: app/src/main/java/com/example/whattodo/WhatToDo.kt
package com.example.whattodo

import android.app.Application
import android.app.NotificationChannel
import android.app.NotificationManager
import android.os.Build

class WhatToDo : Application() {
    override fun onCreate() {
        super.onCreate()
        createNotificationChannels()
    }

    private fun createNotificationChannels() {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
            val mainChannel = NotificationChannel(
                    MAINCHANNEL_ID,
                    "Main Chanel",
                    NotificationManager.IMPORTANCE_HIGH
            )
            mainChannel.description = "This is MainChannel"
            val manager = getSystemService(NotificationManager::class.java)
            manager.createNotificationChannel(mainChannel)
        }
    }

    companion object {
        const val MAINCHANNEL_ID = "Main Chanel"
    }
}